<?xml version="1.0"?>
<!--orb_slam2单目-->
<launch>
    <arg name="depth_camera_name"   default="depth_cam"/>

    <!--深度摄像头-->
    <include file="$(find hiwonder_peripherals)/launch/depth_cam.launch">
        <arg name="depth_camera_name" value="$(arg depth_camera_name)"/>
    </include>

    <include file="$(find hiwonder_servo_controllers)/launch/start.launch"/>
    <node name="init_pose" pkg="hiwonder_slam" type="init_pose.py" output="screen">
        <param name="horizontal" value="true"/>
    </node>

    <node name="orb_slam2_mono" pkg="ORB_SLAM2" type="Mono" output="screen" args="$(find ORB_SLAM2)/../../../Vocabulary/ORBvoc.txt $(find ORB_SLAM2)/../../Monocular/TUM1.yaml">
        <remap from="/camera/image_raw"     to="/$(arg depth_camera_name)/rgb/image_raw" />
        <remap from="/camera/camera_info"   to="/$(arg depth_camera_name)/rgb/camera_info" />
    </node>
</launch>
